addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18549,7 +18549,12 @@ msgctxt "#36012"
 msgid "Couldn't initialise the CEC adapter. Please check your settings."
 msgstr ""
 
-#empty strings from id 36013 to 36014
+#: system/peripherals.xml
+msgctxt "#36013"
+msgid "Represent type of device"
```

Comment:
Maybe "Type of CEC device"?

Reply:
I am not sure. It's about the CEC client (libCEC) what it should "represent" on the CEC bus.

Only "Type of device" sounds for me worded too vaguely. What type, and what type of whom?

Reply:
Good point. What about "CEC mode"?

Reply:
Or "Type of CEC device"? Either way the verb should be dropped, and the label made only a grammatical subject.

Reply:
"CEC client device mode"? I think "device" should be included as it represent a type of device like tuner, recorder or player.

Reply:
I like "CEC client device mode"